test_cli.py

```diff
@@ -1045,14 +1042,33 @@ def test(ctx):
     wait(sugared_remote_uuid)
     check_contains(sugared_remote_name, _run_command([cl, 'search', sugared_remote_name]))
     check_equals(sugared_remote_uuid, _run_command([cl, 'search', sugared_remote_name, '-u']))
-    check_equals('ready', _run_command([cl, 'info', '-f', 'state', sugared_remote_uuid]))
     check_equals('hello', _run_command([cl, 'cat', sugared_remote_uuid + '/stdout']))
 
     # Explicitly fail when a remote instance name with : in it is supplied
     _run_command(
         [cl, 'run', 'cat %%%s//%s%%/stdout' % (source_worksheet_full, name)], expected_exit_code=1
     )
 
+    # Multi-alias tests
+    multi_alias_uuid1 = _run_command(
+        [cl, 'run', 'foo:{}'.format(uuid), 'foo1:{}'.format(uuid), 'echo "two aliases"']
+    )
+    wait(multi_alias_uuid1)
+    check_equals('two aliases', _run_command([cl, 'cat', multi_alias_uuid1 + "/stdout"]))
+
+    multi_alias_uuid2 = _run_command(
+        [
+            cl,
+            'run',
+            'foo:{}'.format(uuid),
+            'foo1:{}'.format(uuid),
+            'foo2:{}'.format(uuid),
+            'echo "three aliases"',
```

Comment:
It would be a nicer test if we `cat foo foo1 foo2` and make sure that they actually have the correct contents.  I think we don't need three aliases if we just have two.

Reply:
Previously, two aliases worked well during my local testing. But three aliases one failed without the changes that I added later on. So I guess I'll just keep the three aliases test to be safe.